development/pubsub-connector/pubSubGateway/README.md

```diff
@@ -0,0 +1,31 @@
+# PubSub Gateway
+
+## Overview
+
+The PubSub Gateway is written in Golang. It pulls messages from PubSub, converts them into a CloudEvents compliant events and forwards them to the Kyma Event Publisher Proxy.
+
+## Prerequisites
+
+PubSub Gateway is meant to run in a Pod within the Kyma Runtime. To get one, check installation instructions [Installation guides](https://kyma-project.io/docs/root/kyma#installation-installation) for details.
+
+## Installation
+
+Use a [Dockerfile](Dockerfile) to build a Docker image:
+
+1. `git clone git@github.com:kyma-project/test-infra.git`
+2. `cd test-infra/development/pubsub-connector/pubSubGateway`
+3. `docker build .`
+4. `docker push`
+
+## Usage
+
+PubSub Gateway uses environment variables to read the configuration.
+
+| Environment variable name | Required | Description |
+|----------------|----------|-------------|
+| **APP_NAME** | Yes | PubSub connector application as set in Compass |
+| **PUBSUB_GATEWAY_NAME** | Yes | PubSub Gateway instance name. It will be used as a cloud event sourceID. |
```

Comment:
```suggestion
| **PUBSUB_GATEWAY_NAME** | Yes | PubSub Gateway instance name. It will be used as a Cloud Event sourceID. |
```

Reply:
Changed to CloudEvents as this is the name of spec.